x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/prebuilt_rules/logic/get_rules_to_update.test.ts

```diff
@@ -365,110 +164,3 @@ describe('filterInstalledRules', () => {
     expect(shouldUpdate).toEqual(true);
   });
 });
-
-describe('mergeExceptionLists', () => {
```

Comment:
these unit tests seem good to keep around - `mergeExceptionLists` doesn't require complex service mocks

Reply:
added back in